lld/COFF/DriverUtils.cpp

```diff
@@ -845,13 +845,14 @@ MemoryBufferRef LinkerDriver::convertResToCOFF(ArrayRef<MemoryBufferRef> mbs,
 
 // Create OptTable
 
+#define OPTTABLE_STR_TABLE_CODE
```

Comment:
Is it feasible to split out all the tool driver updates, like refactor all these PREFIX definitions into the table-generated file guarded by `OPTTABLE_STR_TABLE_CODE`, and then make your changes on top of that?

Reply:
I have no idea. It took a lot of work to get even this to work, figuring out each of the different ways that these macros fail.

Unless its impossible to get this reviewed as-is, I'd really rather not add an intermediate state that is entirely artificial compared to what is here...

Reply:
I convinced myself this is not worth it. The changes to the `PrecomputedOptTable` are hard to abstract away. It would be easy to move the PREFIX macro definition here into the table generated file separately from the string table offset change, but we'd still have to touch all these files.